index.php

```diff
@@ -11,7 +11,7 @@
 
 <head>
     <?php
-require_once('HTMLhead.php');
+include_once('HTMLhead.php');
```

Comment:
sorry this sint what i meant. I mean put the file into a folder called include. so your code is:
`require_once('includes/HTMLhead.php');`

Reply:
oh right ok